### DISCUSSION TITLE
请问图片中 file 是网址的怎么处理的

### DISCUSSION THREAD

**Discussion by Arondight** [takayama-lily/oicq#302] [Q&A] [answered]
```
{
  type: 'image',
  file: 'https://xxx.png',
}
```

请问图片 CQ 里面，file 是网址的，oicq 会先把文件下载到本地然后发过去呢，还是直接把网址发过去让腾讯处理呢？

@takayama-lily

**Comment by Arondight** [comment]
另外这个网址是只有填腾讯家的才可以还是任意一个可以访问的 url 都行

**Comment by takayama-lily** [comment, marked as answer]
下载到本地

**Reply by Arondight** [reply]
谢谢

**Comment by PluginsKers** [comment]
> ```
> {
>   type: 'image',
>   file: 'https://xxx.png',
> }
> ```
> 
> 请问图片 CQ 里面，file 是网址的，oicq 会先把文件下载到本地然后发过去呢，还是直接把网址发过去让腾讯处理呢？
> 
> @takayama-lily 

oicq不会对url进行处理，而目前也不支持直接发送url

如果需要发送。。。

- 使用`axios`或者其他的方法下载到本地
> 这边同样是可以发送base64的，传参和你上面写的一样

**Reply by Arondight** [reply]
谢谢，我是看例子这么写的，问问直接发送任意 url 的可能性
